lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -172,6 +172,23 @@ splitAndAppendTarget(NamedAttrList &annotation, StringRef target,
   return targetPair;
 }
 
+struct ExpandedAndSplitNonLocal {
+  SmallVector<std::tuple<std::string, std::string, std::string>> nlaTargets;
+  std::string leafTarget;
+  llvm::Optional<ArrayAttr> leafSubfields;
+};
+
+// Calls `expandNonLocal` and `splitAndAppendTarget` and combines their results
+// in a struct.
+ExpandedAndSplitNonLocal expandAndSplitNonLocal(StringRef target,
```

Comment:
I'm not up to speed on this code, so I can't provide a useful review, but I'd recommend marking this function `static`